_states/jenkins_plugin.py

```diff
@@ -0,0 +1,119 @@
+import logging
+logger = logging.getLogger(__name__)
+
+install_plugin_groovy = """\
+import jenkins.model.*
+import java.util.logging.Logger
+
+def logger = Logger.getLogger("")
+def installed = false
+def exists = false
+def initialized = false
+def pluginName="{plugin}"
+def instance = Jenkins.getInstance()
+def pm = instance.getPluginManager()
+def uc = instance.getUpdateCenter()
+uc.updateAllSites()
```

Comment:
Heads up: how to run update only once per state execution to avoid updating separately for each plugin?
And it should probably run update only after check if plugin isn't already installed (to speed up next runs).